tests/integration/helpers.py

```diff
@@ -155,7 +159,7 @@ async def get_alertmanager_groups(ops_test: OpsTest, unit_name, unit_num, retrie
     """
     # TODO consume this from alertmanager_client when becomes available
     address = await get_unit_address(ops_test, unit_name, unit_num)
-    url = f"http://{address}:9093/api/v2/alerts/groups"
+    url = f"http://{address}:9093{path}/api/v2/alerts/groups"
```

Comment:
We can use the same here:


```suggestion
    url = f"http://{address}:9093/" + str(Path(path.strip("/")) / "api" / "v2" / "alerts" / "groups")
```